Objects/call.c

```diff
@@ -1320,6 +1320,10 @@ _PyStack_UnpackDict(PyObject *const *args, Py_ssize_t nargs, PyObject *kwargs,
         return 0;
     }
 
+    if (PyArg_ValidateKeywordArguments(kwargs) == 0) {
```

Comment:
This change may or may not make sense, but in any case I think that it needs to justified and a test added, preferably on a separate PR.

Reply:
The comment in `Include/cpython/abstract.h` says that `_PyStack_UnpackDict` does not check the type of keyword arguments. So this change is inconsistent with that. If you think that this should be changed, please open a separate issue explaining the rationale for this change.

Reply:
The additional validation should have been in `PyVectorcall_Call` which does need to do validation. I've moved it there.

Reply:
I still think that it's better to move that change to a different PR with rationale explained for this change.